Sources/MockoloFramework/Models/MethodModel.swift

```diff
@@ -81,7 +82,20 @@ final class MethodModel: Model {
         return ret
     }()
     
-    
+    lazy var argsHistory: ArgumentsHistoryModel? = {
+        if isInitializer {
+            return nil
+        }
+
+        let ret = ArgumentsHistoryModel(name: name,
+                                        genericTypeParams: genericTypeParams,
+                                        params: params,
+                                        isHistoryAnnotated: historyCapturedFuncs.contains(name),
```

Comment:
if it doesn't contain, it should return nil instead.

Reply:
Whether the function is annotated or not, the arguments captor must be rendered by `ArgumentsHistoryModel` if `--enable-args-history` is passed.
And it can know in `render(with:encloser:useTemplateFunc:useMockObservable:enableFuncArgsHistory:)`.

So it should return model in this time, isn't it?

Reply:
That's correct. This is fine then.